src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -99,6 +99,12 @@ def shutdown(self):
         """
         raise NotImplementedError('Abstract Method: shutdown')
 
+    def setEnv(self,name, value=None):
+        """Sets environment variables. If no value is provided it will be looked up
```

Comment:
Needs more substance. Describe what the effect of this method on the running jobs is.